src/components/AssetsListRow/AssetsListRow.js

```diff
@@ -121,14 +125,26 @@ export class AssetsListRow extends Component {
               <SmallText>
                 Current Price:
                 <GreenText>
-                  {currencyFormat(marketData?.current_price.usd, '$')}
+                  {currencyFormat(
+                    marketData?.price_change_24h_in_currency?.[currency],
+                    currencySymbol
+                  )}
                 </GreenText>
               </SmallText>
               <SmallText>
                 Price Change 24h:
-                <GreenText price={marketData?.price_change_24h.toFixed(2)}>
-                  {directionIndicator(marketData?.price_change_24h)}$
-                  {marketData?.price_change_24h.toFixed(2)}
+                <GreenText
+                  price={marketData?.price_change_24h_in_currency?.[
+                    currency
+                  ].toFixed(2)}
+                >
+                  {directionIndicator(
+                    marketData?.price_change_24h_in_currency?.[currency]
```

Comment:
you are using `marketData?.price_change_24h_in_currency?.[currency]` 4 times, better extract this in a variable